website/pages/docs/advanced-topics/monitoring.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Monitoring CloudQuery
+description: Short walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs
+---
+
+# Monitoring
+
+Monitoring CloudQuery can be done in a number of main ways:
+
+- Logging
+- OpenTelemetry
+
+## Logging
+
+CloudQuery is utilizing structured [logging](../reference/cli/cloudquery) (in plain and json formats) which can be analyzed by local tools such as `jq`, `grep` and remote aggregations tools like `loki`, `datadog` or any other popular log aggregation that supports structured logging.
+
+## OpenTelemetry (Preview)
+
+ELT workloads can be long running and sometimes it is necessary to better understand what calls are taking the most time; to potentially optimize those on the plugin side, ignore them or split them to a different workload.
+
+CloudQuery supports [OpenTelemetry](https://opentelemetry.io/) tracing out of the box and can be enabled easily via [configuration](../reference/source-spec).
```

Comment:
```suggestion
CloudQuery supports [OpenTelemetry](https://opentelemetry.io/) tracing out of the box and can be enabled easily via [configuration](/docs//reference/source-spec).
```

We usually have absolute links so if we move this page to another location the link still works